src/query/functions/temporal/aggregation.go

```diff
@@ -85,28 +92,42 @@ func NewAggOp(args []interface{}, optype string) (transform.Params, error) {
 			aggFunc: aggregationFunc,
 		}
 
+		if optype == QuantileType {
+			if len(args) != 2 {
+				return emptyOp, fmt.Errorf("invalid number of args for %s: %d", QuantileType, len(args))
+			}
+
+			scalar, ok := args[0].(float64)
+			if !ok {
+				return emptyOp, fmt.Errorf("unable to cast to scalar argument: %v for %s", args[1], QuantileType)
+			}
+
+			a.quantileScalar = scalar
+		}
+
 		return newBaseOp(args, optype, a)
 	}
 
 	return nil, fmt.Errorf("unknown aggregation type: %s", optype)
 }
 
 type aggNode struct {
-	op         baseOp
-	controller *transform.Controller
-	aggFunc    func([]float64) float64
+	op             baseOp
+	controller     *transform.Controller
+	aggFunc        func([]float64, float64) float64
+	quantileScalar float64
```

Comment:
Do we have to add this quantileScalar at the generic agg func layer? Doesn't seem scalable given how many different types of agg funcs we can have. Can we just do the args checking and parsing once we get to the functions that need them?